### DISCUSSION TITLE
Welcome to lit Discussions!

### DISCUSSION THREAD

**Discussion by justinfagnani** [lit/lit#2082] [Announcements]
## 👋 Welcome!

We have enable GitHub Discussions to create another space for the Lit community to connect in addition to our [Slack workspace](https://lit.dev/slack-invite/) and [Twitter](https://twitter.com/buildWithLit), and to separate questions and ideas from GitHub issues.

We think that discussions will be good for long-form questions and be more discoverable when a topic is of interest to many community members of time. Chat and social media tend to be hard to search and ephemeral. Discussions should be more persistent. GitHub discussions are also great for those who aren't on Slack and Twitter!

We hope that you will:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

Remember, all interactions must adhere to our [code of conduct](https://github.com/lit/lit/blob/main/CODE_OF_CONDUCT.md). Please be kind!

**Comment by abdonrd** [comment]
Great! I just created a PR to encourage its use: https://github.com/lit/lit/pull/2083

**Comment by net43657** [comment]
hi
